openoa/utils/machine_learning_setup.py

```diff
@@ -107,6 +107,9 @@ class MachineLearningSetup:
     random_search: Any = field(init=False)
     opt_hyp: Any = field(init=False)
     opt_model: Any = field(init=False)
+    n_jobs: int | None = field(
```

Comment:
Since n_jobs is only used as an argument in the `hyper_optimize` method, is there a reason to list it as a class attribute here? If so, should this be described in the docstring, indicating that it can be specified when initializing an object of the class.